commandlineparser/src/Parser.cpp

```diff
@@ -52,5 +54,25 @@ void  cmdParser::Parser::tokenizer(int argc, char*argv[])
 bool cmdParser::Parser::process_data(int argc, char*argv[])
 {
 	tokenizer(argc,argv);
+	if (argc == 1)
+	{
+		default_help();
+		//To terminate the program.
+		std::exit(0);
+	}
 	return true;
+}
+
+void cmdParser::Parser::default_help() const
+{
+	std::cout << "\n **************WELCOME TO OPTIONS OF THE LIBRARY *********************" << std::endl;
+	std::cout << "[FORMAT]" << std::endl;
+	std::cout << "-------------------------------------------------------------------------------------" << std::endl;
+	std::cout << "Short_Command" << "\t" << "Long_Command" << "\t" << "Short_Description" << "\t\t" << "Long_Description" << std::endl;
+	std::cout << "-------------------------------------------------------------------------------------" << std::endl;
+	for (auto it = help_command_list.begin(); it != help_command_list.end(); ++it)
```

Comment:
use range based for loop

Reply:
Thanks for the nice suggestion, but implemented using std::for_each();